python/tvm/build_module.py

```diff
@@ -379,24 +379,40 @@ def lower(sch,
         return stmt
     return ir_pass.MakeAPI(stmt, name, arg_list, 0, cfg.restricted_func)
 
+
 def build(sch,
           args=None,
           target=None,
           target_host=None,
           name="default_function",
-          binds=None,
-          postpone_host_codegen=False):
+          binds=None):
     """Build a function with arguments as signature. Code will be generated
-    for a device specified by the target. For homogeneous execution, a module
-    that contains both host and device code is returned. For heterogeneous
-    execution, a list of lowered functions for the host and a module containing
-    device code are returned, but actual code generation for the host module is
-    postponed after code generation is finished for all devices.
+    for devices coupled with target information.
+
+    There are two typical uses of this functioin:
+        1. For backward compatibility
```

Comment:
This is a valid case of usage so we don't want to see for backward compatiblity.

Consider put these code examples under Example section with rst real code block